src/test/java/uk/gov/pay/ledger/agreement/resource/AgreementResourceIT.java

```diff
@@ -56,7 +59,10 @@ public void shouldGetAgreementWithPaymentInstrument() {
                 .statusCode(Response.Status.OK.getStatusCode())
                 .contentType(JSON)
                 .body("external_id", is(agreementFixture.getExternalId()))
-                .body("payment_instrument.external_id", is(paymentInstrumentFixture.getExternalId()));
+                .body("payment_instrument.external_id", is(paymentInstrumentFixture.getExternalId()))
+                .body("payment_instrument.type", is(PaymentInstrumentType.CARD.name()))
+                .body("payment_instrument.card_details.card_type", is(CardType.CREDIT.name().toLowerCase()))
```

Comment:
Really this should probably use `.toLowerCase(Locale.ENGLISH)` or `.toLowerCase(Locale.ROOT)` rather than relying on the default locale.

Reply:
Good catch, we've got a few follow up PRs so will include this in those